experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -691,6 +691,14 @@ export interface OptionalField<TTypes extends AllowedTypes> extends TreeField {
 
 // #region Typed
 
+/**
+ * The strongly typed {@link Tree} API for a given schema.
+ * @alpha
+ */
+export type Typed<TSchema extends FieldSchema | TreeSchema> = TSchema extends FieldSchema
+	? TypedField<TSchema>
+	: TypedNode<Assume<TSchema, TreeSchema>>;
+
```

Comment:
Unused?

Reply:
oops, didn't intent to include that. Fixed.